src/test/java/com/epam/tc/hw7/dataprovider/DataProviderForMetalsColorsTest.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.tc.hw7.dataprovider;
+
+import com.epam.tc.hw7.reader.JsonReaderForDatas;
+import org.testng.annotations.DataProvider;
+
+public class DataProviderForMetalsColorsTest {
+
+    @DataProvider(name = "data for metals color test")
+    public static Object [][] dataProviderForMetalsColorsTest() {
+        return new Object[][] {
```

Comment:
One of the HW condition: DO NOT fix the number of rows in test file. 
It should work file with only data_1 and also with data_1.... data_20.
This function should be changed